src/main/app/claims/models/response/core/courtDetermination.ts

```diff
@@ -0,0 +1,9 @@
+import { PaymentIntention } from 'claims/models/response/core/paymentIntention'
+
+export class CourtDetermination {
+
+  courtDecision: PaymentIntention
```

Comment:
I think changing variable name to courtPaymentIntention or something else more appropriate may make more sense!

Reply:
it was renamed to `courtDecision` in `claim-store-api` after discussion. please do not change it.